nova/virt/vmwareapi/vmops.py

```diff
@@ -1987,12 +1987,9 @@ def _resize_vm(self, context, instance, vm_ref, flavor, image_meta):
                                                             self._cluster,
                                                             vm_ref,
                                                             operation='remove')
-            except vexc.VimFaultException as e:
-                with excutils.save_and_reraise_exception() as ctx:
-                    if 'InvalidArgument' in e.fault_list:
-                        LOG.debug('DRS override was already deleted.',
-                                  instance=instance)
-                        ctx.reraise = False
+            except Exception:
+                LOG.exception('Could not remove DRS override.',
+                              instance=instance)
```

Comment:
Is the `instance` kwargs even used? The message string has no format.

Reply:
Yes. It's in Nova's internal log format. You can see that nearly all log messages provide this parameter.